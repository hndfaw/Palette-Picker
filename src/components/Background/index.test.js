import React from 'react';
import {Background} from './index.jsx';
import { shallow } from 'enzyme';



describe('Background', () => {
  it('It should match the snapshot', () => {

    const wrapper = shallow(<Background />)
    expect(wrapper).toMatchSnapshot();
  })
})