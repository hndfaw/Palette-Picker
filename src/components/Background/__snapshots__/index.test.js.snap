// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Background It should match the snapshot 1`] = `ShallowWrapper {}`;
